### PR TITLE
Jameslotery/dev 2146

### PR DESCRIPTION
**Pull request by jlotery**
Check that the email address sent to the billing AP is a valid email before submitting to stripe. If invalid return a 400 error response.

### CODE REVIEW

File: services/stripe-billing/lib/index.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
 /* eslint-disable no-prototype-builtins */
 const CampsiService = require('../../../lib/service');
 const helpers = require('../../../lib/modules/responseHelpers');
+const { hasUncaughtExceptionCaptureCallback } = require('process');
```
Comment (gabila-r) [inline services/stripe-billing/lib/index.js:4]: Why did you require this ? 🤔

File: services/stripe-billing/lib/index.js [hunk 2]
```diff
@@ -74,7 +76,13 @@ module.exports = class StripeBillingService extends CampsiService {
     });
 
     this.router.post('/customers', (req, res) => {
-      stripe.customers.create(bodyToCustomer(req.body, 'source', req.user), defaultHandler(res));
+      try{
```
Comment (gabila-r) [inline services/stripe-billing/lib/index.js:79]:
```suggestion
      try {
```
Linter not trigger ?

File: services/stripe-billing/lib/index.js [hunk 3]
```diff
@@ -83,7 +91,13 @@ module.exports = class StripeBillingService extends CampsiService {
     });
 
     this.router.put('/customers/:id', (req, res) => {
-      stripe.customers.update(req.params.id, bodyToCustomer(req.body, 'default_source'), defaultHandler(res));
+      try{
```
Comment (gabila-r) [inline services/stripe-billing/lib/index.js:94]:
```suggestion
      try {
```
Same

**Review by gabila-r — CHANGES_REQUESTED** [review]
Little update to do :grim

File: services/stripe-billing/lib/index.js (hunk 1 above)
Comment (jlotery) [reply services/stripe-billing/lib/index.js:4]: oops, my vscode editor added it when the autocompletion feature wrongly added that object :( I'll remove it . thx :)

**Review by gabila-r — APPROVED** [review]

**Pull request merged**